docs/apps/rbac.md

```diff
@@ -0,0 +1,538 @@
+
+With all 3 ids ready, POST to http://127.0.0.1:8000/api/v1/role_user_assignments/
+
+```json
+{
+    "role_definition": 3,
+    "object_id": 3,
+    "user": 3
+}
+```
+
+This will give user id=3 view permission to a single inventory id=3, assuming the role definition
+referenced is what was created in the last section.
+
+### Assigning a User as a Member of a Team
+
+While this is possible with the RBAC API, it is not covered here,
+because it may come from an external source.
+
+The "member_team" permission may later be prohibited from use in custom role definitions.
+
+### Assigning a Team a Role to an Object
+
+If you used the test_app bootstrap script, then you will find the user "angry_spud"
+is a member of the "awx_devs" team.
+
+Assuming the team id is 2, POST to
+
+http://127.0.0.1:8000/api/v1/role_team_assignments/
+
+with data
+
+```json
+{
+    "role_definition": 3,
+    "object_id": 3,
+    "team": 2
+}
+```
+
+This has a similar effect to user assignments, but in this case will give
+permissions to all users of a team
+
+### Viewing Assignments
+
+For the single inventory mentioned above, it is possible to view the existing permission
+assignments directly to that object.
+
+GET
+
+http://127.0.0.1:8000/api/v1/role_team_assignments/?object_id=3&content_type__model=inventory
+
+http://127.0.0.1:8000/api/v1/role_user_assignments/?object_id=3&content_type__model=inventory
+
+### Revoking an Assignment
+
+From any of the assignment lists, the user can select an assignment to revoke.
+Follow the "url" in the serializer from either user or team assignment lists.
+
+DELETE http://127.0.0.1:8000/api/v1/role_user_assignments/1/
+
+Will undo everything related to that assignment.
+The user or team's users will lose permission that was granted by the object-role-assignment.
+
+### Assigning Organization Permission
+
+In the case of inventory, its parent object is its organization.
+Instead of giving permission to a single inventory, you can use roles to give
+permission to all inventories inside of an organization.
+
+The difference in the above steps are that
+- when creating a custom role definition the `content_type` would be "shared.organization"
+- when creating the assignment, the `object_id` would be the id of an organization
+
+### Displaying Access
+
+To give a complete picture of access, consider that there are 3 "layers" of access.
+ - Direct object role assignment - for users and teams
+ - Parent object role assignment - for users and teams
+ - Superuser or super-auditor user flags
+
+For a single app using this system at the org-level, these 5 distinct pieces
+of information would give a complete summary of who has access.
+By itself, this does not expand the users who are members of teams.
+Assuming that teams-in-teams are enabled, this would not just be the direct
+members of the teams, but a listing of all users who have indirect membership to the team.
```

Comment:
Do we have any endpoints that will give this to us or is this something that just has to be computed?

Reply:
No https://github.com/ansible/django-ansible-base/issues/150, but I want to make the point that _this_ system with these models is actively working to produce the content of the _old_ access list in AWX in its corresponding PR.

Reply:
I removed this section, because we have an issue and this is unmaintainable.